foreman/data_refinery_foreman/surveyor/management/commands/start.py

```diff
@@ -1,4 +1,4 @@
-from django.core.management.base import BaseCommand, CommandError
+from django.core.management.base import BaseCommand
```

Comment:
The reason was because this is an end-to-end test and I wasn't sure how to do that yet. It seems like it wouldn't be too hard to do though: http://docs.celeryproject.org/en/latest/userguide/testing.html.

I would agree that I should add them in this PR however I remembered that the next PR I am going to start will be significantly changing this downloader. Therefore rather than writing a bunch of tests now that I know will become obsolete in the very near future.